projects/plugins/protect/src/js/routes/home/home-statcards.jsx

```diff
@@ -0,0 +1,243 @@
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { useMemo } from 'react';
+import useScanStatusQuery, { isScanInProgress } from '../../data/scan/use-scan-status-query';
+import usePlan from '../../hooks/use-plan';
+import useWafData from '../../hooks/use-waf-data';
+import styles from './styles.module.scss';
+
+const IconWithLabel = ( { label, isSmall, icon } ) => (
+	<span className={ styles[ 'stat-card-icon' ] }>
+		{ icon }
+		{ ! isSmall && (
+			<Text className={ styles[ 'stat-card-icon-label' ] } variant="body-extra-small">
+				{ label }
+			</Text>
+		) }
+	</span>
+);
+
+const HomeStatCard = ( { text, args } ) => (
+	<Tooltip className={ styles[ 'stat-card-tooltip' ] } text={ text }>
+		<div className={ styles[ 'stat-card-wrapper' ] }>
+			<StatCard { ...args } />
+		</div>
+	</Tooltip>
+);
+
+const HomeStatCards = () => {
+	const { hasPlan } = usePlan();
+	const [ isSmall ] = useBreakpointMatch( [ 'sm', 'lg' ], [ null, '<' ] );
+	const variant = isSmall ? 'horizontal' : 'square';
+
+	const { data: status } = useScanStatusQuery();
+	const scanning = isScanInProgress( status );
+	const numThreats = status.threats.length;
+	const scanError = status.error;
+
+	let lastCheckedLocalTimestamp = null;
+	if ( status.lastChecked ) {
+		// Convert the lastChecked UTC date to a local timestamp
+		lastCheckedLocalTimestamp = dateI18n(
+			'F jS g:i A',
+			new Date( status.lastChecked + ' UTC' ).getTime(),
+			false
+		);
+	}
+
+	const {
+		config: { bruteForceProtection: isBruteForceModuleEnabled },
+		isEnabled: isWafModuleEnabled,
+		wafSupported,
+		stats,
+	} = useWafData();
+
+	const {
+		blockedRequests: { allTime: allTimeBlockedRequestsCount = 0 } = {},
+		blockedLogins: allTimeBlockedLoginsCount = 0,
+	} = stats || {};
+
+	const iconHeight = useMemo( () => 20, [] );
+
+	const lastCheckedMessage = useMemo( () => {
+		if ( scanError ) {
+			return __(
+				'Please check your connection or try scanning again in a few minutes.',
+				'jetpack-protect'
+			);
+		}
+
+		if ( scanning ) {
+			return __( 'Your results will be ready soon.', 'jetpack-protect' );
+		}
+
+		const entityLabel = hasPlan
+			? _n( 'threat', 'threats', numThreats, 'jetpack-protect' )
+			: _n( 'vulnerability', 'vulnerabilities', numThreats, 'jetpack-protect' );
+
+		if ( lastCheckedLocalTimestamp ) {
+			if ( numThreats > 0 ) {
+				return sprintf(
+					// translators: %1$s: date/time, %2$d: number, %3$s: entity label
+					__( 'Last checked on %1$s: We found %2$d %3$s.', 'jetpack-protect' ),
+					lastCheckedLocalTimestamp,
+					numThreats,
+					entityLabel
+				);
+			}
+			return sprintf(
+				// translators: %s: date/time
+				__( 'Last checked on %s: Your site is secure.', 'jetpack-protect' ),
+				lastCheckedLocalTimestamp
+			);
+		}
+
+		return sprintf(
+			// translators: %1$s: date/time, %2$d: number, %3$s: entity label
+			__( 'Last scan we found %2$d %3$s.', 'jetpack-protect' ),
+			lastCheckedLocalTimestamp,
+			numThreats,
+			entityLabel
+		);
```

Comment:
We should avoid concatenating translated strings together.

See FG: PCYsg-3hI-p2#it-s-ok-to-almost-repeat-yourself

To avoid this, we will need to repeat some of the strings, i.e. `hasPlan ? __( 'We identified %s threats' ) : __( 'We identified %s vulnerabilities' )`

Reply:
Thanks for the link, good to know! I have updated to avoid that approach in [5728962](https://github.com/Automattic/jetpack/pull/40317/commits/57289626b75a733d1cd92c7ff0543b97f725f823) - its a little more complicated now since we needed to consider `hasPlan` for `threat` vs `vulnerability` _and_ `numThreats` for plurals. Will keep this all in mind moving forward.